Jellyfin.Api/Controllers/UniversalAudioController.cs

```diff
@@ -88,16 +88,14 @@ public class UniversalAudioController : BaseJellyfinApiController
         /// <response code="302">Redirected to remote audio stream.</response>
         /// <returns>A <see cref="Task"/> containing the audio file.</returns>
         [HttpGet("Audio/{itemId}/universal")]
-        [HttpGet("Audio/{itemId}/universal.{container}", Name = "GetUniversalAudioStream_2")]
         [HttpHead("Audio/{itemId}/universal", Name = "HeadUniversalAudioStream")]
-        [HttpHead("Audio/{itemId}/universal.{container}", Name = "HeadUniversalAudioStream_2")]
         [Authorize(Policy = Policies.DefaultAuthorization)]
         [ProducesResponseType(StatusCodes.Status200OK)]
         [ProducesResponseType(StatusCodes.Status302Found)]
         [ProducesAudioFile]
         public async Task<ActionResult> GetUniversalAudioStream(
             [FromRoute, Required] Guid itemId,
-            [FromRoute] string? container,
```

Comment:
This container is a set of audio codecs to create the directplay profiles. It's from query instead of route.

https://github.com/jellyfin/jellyfin-web/blob/f19b6f9a2a8b2da7485ae3126079cdc1c1ff89a5/src/components/playback/playbackmanager.js#L314